tests/sentry/snuba/metrics/test_utils.py

```diff
@@ -0,0 +1,265 @@
+from datetime import datetime
+
+import pytest
+
+from sentry.snuba.metrics.utils import get_intervals, get_num_intervals
+
+MINUTE = 60
```

Comment:
Are you also planning to test cases with `MINUTES`?

Reply:
We are using `MINUTE`

```
HOUR =  60 * MINUTE
```

And HOUR is used below.

`DAY` is not used at the moment but I wouldn't remove it.

Reply:
What I meant was making some tests with `granularity, interval` with `MINUTE`

Reply:
Oh, got it :-).
No I was not planning to test minute granularity, but why not.
I'll and test cases for this as well.